RenderStateNotationParser/src/RenderStateNotationParserImpl.cpp

```diff
@@ -26,131 +26,154 @@
 
 #include "pch.h"
 
-#include "RenderStateNotationParserImpl.hpp"
-
 #include <unordered_set>
+#include <functional>
 
 #include "FileWrapper.hpp"
 #include "DataBlobImpl.hpp"
 #include "DefaultRawMemoryAllocator.hpp"
+#include "RenderStateNotationParserImpl.hpp"
 
 namespace Diligent
 {
 
-static void Deserialize(const nlohmann::json& Json, PipelineStateNotation& Type, DynamicLinearAllocator& Allocator)
+struct InlineStructureCallbacks
```

Comment:
This struct and all static functions are local to this file, they should be in a nameless namespace:

```cpp
namespace Diligent
{

namespace
{
...
```